libindex/libindex.go

```diff
@@ -186,7 +186,7 @@ func (l *Libindex) IndexReport(ctx context.Context, hash claircore.Digest) (*cla
 }
 
 // AffectedManifests retrieves a list of affected manifests when provided a list of vulnerabilities.
-func (l *Libindex) AffectedManifests(ctx context.Context, vulns []claircore.Vulnerability) (claircore.AffectedManifests, error) {
+func (l *Libindex) AffectedManifests(ctx context.Context, vulns []claircore.Vulnerability) (*claircore.AffectedManifests, error) {
```

Comment:
looks good, but was expecting to see some changes in the notifier code, not necessary? Auto de-ref may take care of it without changes.

Reply:
What notifier code? This is only used in the libvuln HTTP handler, as far as I can tell.

Reply:
Sorry, I meant I was expecting to see a linked pr: notifier in Clair utilizes this: https://github.com/quay/clair/blob/main/notifier/processor.go